llvm/test/tools/llvm-symbolizer/no-use-symbol-table.s

```diff
@@ -0,0 +1,219 @@
+# REQUIRES: x86-registered-target
+
+# RUN: llvm-mc -filetype=obj -triple=x86_64-pc-linux %s -o %t.o
+# RUN: llvm-addr2line --no-use-symbol-table -fe %t.o 0x1 | FileCheck %s
```

Comment:
Prefer to use the expanded forms of the other commands, as a casual user might think `-fe` is a specific command and wonder what it's for.